Description: disable samtools pileup commands in the example tests.
 ‘samtools pileup’ has been dropped in version 0.1.17.
Forwarded: samtools-devel@lists.sourceforge.net
Origin: vendor

--- a/examples/Makefile
+++ b/examples/Makefile
@@ -1,5 +1,5 @@
 all:../libbam.a ../samtools ../bcftools/bcftools \
-	ex1.glf ex1.pileup.gz ex1.bam.bai ex1f-rmduppe.bam ex1f-rmdupse.bam ex1.glfview.gz ex1.bcf calDepth
+	ex1.bam.bai ex1f-rmduppe.bam ex1f-rmdupse.bam ex1.bcf calDepth
 		@echo; echo \# You can now launch the viewer with: \'samtools tview ex1.bam ex1.fa\'; echo;
 
 ex1.fa.fai:ex1.fa
@@ -8,12 +8,6 @@
 		../samtools import ex1.fa.fai ex1.sam.gz ex1.bam
 ex1.bam.bai:ex1.bam
 		../samtools index ex1.bam
-ex1.pileup.gz:ex1.bam ex1.fa
-		../samtools pileup -cf ex1.fa ex1.bam | gzip > ex1.pileup.gz
-ex1.glf:ex1.bam ex1.fa
-		../samtools pileup -gf ex1.fa ex1.bam > ex1.glf
-ex1.glfview.gz:ex1.glf
-		../samtools glfview ex1.glf | gzip > ex1.glfview.gz
 ex1a.bam:ex1.bam
 		../samtools view -h ex1.bam | awk 'BEGIN{FS=OFS="\t"}{if(/^@/)print;else{$$1=$$1"a";print}}' | ../samtools view -bS - > $@
 ex1b.bam:ex1.bam
